internal/jobcontainers/process.go

```diff
@@ -92,21 +92,30 @@ func (p *JobProcess) Signal(ctx context.Context, options interface{}) (bool, err
 func (p *JobProcess) CloseStdin(ctx context.Context) error {
 	p.stdioLock.Lock()
 	defer p.stdioLock.Unlock()
-	return p.stdin.Close()
+	if p.stdin != nil {
```

Comment:
Can we move this to a separate PR?

Reply:
Sure

Reply:
https://github.com/microsoft/hcsshim/pull/1115